### PR TITLE
fix deploy properties overwrite

### PR DESCRIPTION
**Pull request by RuoyuWang-MS**
fix deploy properties overwrite

### CODE REVIEW

File: src/DeploymentProvider/DeploymentHelper.ts [hunk 1]
```diff
@@ -3,22 +3,50 @@ import { AppPlatformManagementClient, AppPlatformManagementModels as Models } fr
 import { uploadFileToSasUrl } from "./azure-storage";
 import * as core from "@actions/core";
 import { parse } from 'azure-actions-utility/parameterParserUtility';
+import {deploymentName} from "@azure/arm-appplatform/esm/models/parameters";
```
Comment (allxiao) [inline src/DeploymentProvider/DeploymentHelper.ts:6]: space in `{}` to stay consistent with the above lines.

File: src/DeploymentProvider/DeploymentHelper.ts [hunk 2]
```diff
@@ -3,22 +3,50 @@ import { AppPlatformManagementClient, AppPlatformManagementModels as Models } fr
...
     }
 
+    private static async getDeployment(client: AppPlatformManagementClient, params: ActionParameters, deploymentName: string): Promise<Models.DeploymentResource> {
+        if (this.listDeploymentsResult != null) {
+            core.debug('get from list cache, list deployments response: ' + this.listDeploymentsResult._response.bodyAsText);
+            let ret: Models.DeploymentResource;
+            this.listDeploymentsResult.forEach(deployment => {
+                core.debug('deployment str: ' + JSON.stringify(deployment));
+                if (deployment.name == deploymentName) {
+                    ret = deployment;
```
Comment (allxiao) [inline src/DeploymentProvider/DeploymentHelper.ts:37]: just return here?

**Review by allxiao — CHANGES_REQUESTED** [review]
added two minor comments.

File: src/DeploymentProvider/DeploymentHelper.ts (hunk 2 above)
Comment (RuoyuWang-MS) [reply src/DeploymentProvider/DeploymentHelper.ts:36]: https://stackoverflow.com/questions/2641347/short-circuit-array-foreach-like-calling-break
Typescript array forEach cannot break or return and we have at most 2 items right now. So is that OK to loop the array without break?

**Review by allxiao — APPROVED** [review]

**Pull request merged**